index.html

```diff
@@ -268,7 +323,45 @@ <h2 class="javaS">CSS</h2>
           </div>
         </div>
       </div>
-    
-  </section>
+      <footer>
+        <div class="contact-me-img">
+          <img
+            class="contactme-img"
+            src="Microverse-Student-Project-1/Contact.png"
+            alt=""
+          />
+          <img
+            class="contact-shapes"
+            src="Microverse-Student-Project-1/Contact2.png"
+            alt=""
+          />
+        </div>
+        <div class="contact-text">
+          <h1 class="Contactme">Contact Me</h1>
+        </div>
+        <div class="contactmepara">
+          <p class="contactme-p">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
 
-</body>
\ No newline at end of file
+        <form action="#" class="contact-information">
+          <input class="name-input" type="text" name="username" placeholder="alzubair" maxlength="30" />
+          <input
+            class="name-input"
+            type="email"
+            name="usermail"
+            placeholder="youremail@gmail.com"
+            required/>
+          
+          <textarea class="text-box" name="msg" id="message" cols="30" rows="10" maxlength="500">
+            Write your message here.</textarea>
+          
+        </form>
+        <button type="button" class="normal contact-btn">Get in touch</button>
```

Comment:
- [ ] The button should be part of your form with a `type='submit'` to submit the form